lib/src/kv.rs

```diff
@@ -38,6 +46,8 @@ pub enum KvError {
     KeyNotFound,
     #[error("no Tx found")]
     NoTx,
+    #[error("Iterator not found")]
```

Comment:
For all of the `KvError` strings here let's be more consistent in format (same capitalization).